packages/webcomponents/src/api/Analytics.ts

```diff
@@ -54,7 +48,7 @@ class JustifiAnalytics {
   };
 
   private trackCustomEvents() {
-    this.eventEmitters = getEventNames(this.componentInstance);
+    this.eventEmitters = ['submitted', 'error-event'];
```

Comment:
Instead of trying to get the event-names from each component, it's actually easier to use a fixed list here for now. We can wait until we feel the need for a more complex approach.